### ISSUE TITLE
In the 1.17 this won't work

### ISSUE THREAD

**Issue by martinlacorrona** [DataPools/BlacklistSRVBypass#1]
https://minecraft.fandom.com/wiki/Java_Edition_21w13a
See: MC-136551 bug fixed 
![image](https://user-images.githubusercontent.com/9017738/113585368-6d374c00-962c-11eb-995a-b29ddc789a61.png)

Now what guys?
What can we do?

**Comment by martinlacorrona** [comment]
Yes, but I understand that in some moment you will get banned all ips from your server..

**Comment by DataPools** [comment]
Theoretically, someone could make some sort of mod (maybe Fabric?) that could bypass the blacklist by modifying the isBlockedServer() method.

**Comment by martinlacorrona** [comment]
Yes, but in this case you force player to user a different version of the game.

This is the problem, that right now the player didn't need to use other launcher or mods to play blocked servers, but right now... I don't know.
I'm out of ideas.

**Comment by Scifi9902** [comment]
What if you create a modified version of the game that simply always returns true for the isServerBlocked method? (I don't deal with clients much so I may sound braindead here) then they could simply just use the same launcher just a different jar?

**Comment by DataPools** [comment]
Yes, it’s quite a possibility. I believe SquidHQ tried to do just that.

**Comment by LucidAPs** [comment]
Unfraternally the SRV trick doesn't seems to work in 2022.
